src/chapters/modules/wrap-up.rst

```diff
@@ -0,0 +1,15 @@
+Wrap-up
+========
+
+In this chapter, we showed how to use ``require`` to pull a module into your
+project, and we presented two ways to use ``module.exports``. Of course, these
+are not the only ways to share content.
+
+A quick search online shows that besides functions, we can also share
+individual variables. There are also alternative syntaxes for
+``module.exports`` - even one that exports as an object, but imports as a
+function (which means no dot notation). :-O
```

Comment:
The emoji should be eliminated. Screen readers cannot process it.

Reply:
OK. :-(